tls/s2n_cipher_preferences.c

```diff
@@ -82,6 +82,79 @@ const struct s2n_cipher_preferences cipher_preferences_20210831 = {
     .suites = cipher_suites_20210831,
 };
 
+/*
+ * The following security policy is derived from the following specification:
+ * https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-52r2.pdf
+ *
+ * Only supports TLS1.2
+ */
+struct s2n_cipher_suite *cipher_suites_default_fips[] = {
+    /* /1* tls1.3 *1/ */
+    /* /1* TLS_AES_128_GCM_SHA256 (0x13, 0x01) *1/ */
+    /* &s2n_tls13_aes_128_gcm_sha256, */
+    /* /1* TLS_AES_256_GCM_SHA384 (0x13, 0x02) *1/ */
+    /* &s2n_tls13_aes_256_gcm_sha384, */
+    /* /1* TLS_AES_128_CCM_SHA256 (0x13, 0x04) *1/ */
+    /* /1* TLS_AES_128_CCM_8_SHA256 (0x13, 0x05) *1/ */
```

Comment:
Did you mean to leave all these comments in? We should remove code, not comment it out.

Reply:
I have removed the 1.3 commented out policies for now.